src/item/StringToItemParser.php

```diff
@@ -870,6 +870,7 @@ private static function make() : self{
 		$result->registerBlock("stone_wall", fn() => VanillaBlocks::COBBLESTONE_WALL());
 		$result->registerBlock("stonebrick", fn() => VanillaBlocks::STONE_BRICKS());
 		$result->registerBlock("stonecutter", fn() => VanillaBlocks::LEGACY_STONECUTTER());
```

Comment:
I think we can change this to use the new stonecutter as well. Might be worth adding a `legacy_stonecutter` alias for the old one, though.